ui/components/MesheryPatterns.js

```diff
@@ -720,7 +723,7 @@ function PatternForm({ pattern, onSubmit, show }) {
   }
 
   function getPatternAttributeName(jsonSchema) {
-    return PascalCaseToKebab(jsonSchema?._internal?.patternAttributeName || "NA");
+    return jsonSchema?._internal?.patternAttributeName || "NA";
```

Comment:
just curious, is there a reason why we're not using the PascalToKebab function?

Reply:
Actually, pascalToKebab is to search for the keys as a name of service. But that was not serving the purpose since it the name of service may be chosen to be different in most of the ideal scenarios. so now the patterns are matched according to their type, so now type doesns't need this type of transformation.